ruby_test.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Test Hound
+class TestHound
+  NAME = 'Wormbots'
+  VERSION = '0.0.1'
+  LOGISTICS = [NAME, VERSION].freeze
+
+  def foo
+    puts 'bar'
+    puts "foo"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.